tests/testreactive.py

```diff
@@ -768,8 +768,3 @@ def test_reactive_callback_resolve_accessor():
     dfx = rx(df)
     out = dfx["name"].str._callback()
     assert out is df["name"].str
-
-def test_docstrings_in_sync():
```

Comment:
I removed this test because I do believe the docstrings in fact should be different:

Users our `Parameter.rx` would expect a docstring with examples focused on Parameter usage. Users of `rx.rx` property would expect examples focused on `rx` usage.